src/__test__/redux/reducers/__snapshots__/experimentsReducer.test.js.snap

```diff
@@ -188,20 +188,36 @@ Object {
     "id": "experiment-1",
     "lastViewed": "01-01-2021",
     "meta": Object {
+      "backendStatus": Object {
+        "gem2s": Object {
+          "completedSteps": Array [],
+          "endDate": null,
+          "startDate": "2021-01-01T00:00:00.000Z",
+          "status": "SUCCEEDED",
+          "stopDate": "2021-01-01T10:00:00.000Z",
+        },
+        "pipeline": Object {
+          "completedSteps": Array [],
+          "endDate": null,
+          "startDate": "2021-01-01T00:00:00.000Z",
+          "status": "SUCCEEDED",
+          "stopDate": "2021-01-01T10:00:00.000Z",
+        },
+      },
```

Comment:
adding here also the worker state does not really improve testing right?

Reply:
I'm adding that in the next commit, because I don't want to bother ongoing CI, which we'll use for staging. Shouldn't affect anything.